kubernetes/resources_api/overlays/staging/deployment.yaml

```diff
@@ -0,0 +1,14 @@
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: resources-api
+spec:
+  template:
+    spec:
+      containers:
+      - name: app
+        env:
+        - name: FLASK_ENV
+          value: staging
+      - name: postgres
```

Comment:
You can remove the postgres part here, even if you weren't removing the container from this deployment

Reply:
I wasn't sure, so I decided to err on the side of throwing it in there. But now that I'm removing it from the base deployment, I'll definitely remove it from here as well.